src/Core/Services/StateService.cs

```diff
@@ -1234,6 +1234,17 @@ public async Task SetPreLoginEmailAsync(string value)
             await SetValueAsync(Constants.PreLoginEmailKey, value, options);
         }
 
+
+        public ConfigResponse GetConfigs()
+        {
+            return _storageMediatorService.Get<ConfigResponse>(Constants.ConfigsKey);
+        }
+
+        public void SetConfigs(ConfigResponse value)
+        {
+            _storageMediatorService.Save(Constants.ConfigsKey, value);
+        }
```

Comment:
Are there any sensitive data on the config to be saved? These methods (synchronous version) read/write directly to the user preferences/user defaults of the platform.